app-tracking-protection/vpn-impl/src/main/res/drawable/ic_caret_right.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Copyright (c) 2022 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="8dp"
+    android:height="14dp"
+    android:viewportWidth="8"
+    android:viewportHeight="14">
+    <path
+        android:fillAlpha="0.12"
+        android:fillColor="#000000"
```

Comment:
do you want to review colors now or later? if now then we need to make that light / dark theme values for icons are respected.

Reply:
Let's review them later as I updated some of them in `Part 4` and may need some help from you as I don't think they look as expected :sweat_smile: